src/graphql/public/schema.graphql

```diff
@@ -79,6 +80,16 @@ type AccountUpdateDisplayCurrencyPayload {
   errors: [Error!]!
 }
 
+input AccountUpdatePushNotificationSettingsInput {
+  enabled: Boolean!
+  settings: [PushNotificationSettingsInput]!
```

Comment:
I don't quite understand the shape of the input. why is enabled passed at the root?

Reply:
The settings are a combination of "NotificationType" filters that live under `settings` and a general on/off switch for all notifications called `enabled`. Maybe I should change `settings` to be `settingsByNotificationType` or even `filters`.